src/backend/src/transformers/teams.transformer.ts

```diff
@@ -1,19 +1,8 @@
 import { Prisma } from '@prisma/client';
```

Comment:
definitely agree with having a transformers folder for just the transformers, forgot to do that in my pr